monkestation/code/modules/blacksmithing/smithed_items.dm

```diff
@@ -0,0 +1,421 @@
+	finalitem.desc = finaldesc
+	finalitem.name = pick("Delersibnir", "Nekolangrir", "Zanoreshik","Öntakrítin", "Nogzatan", "Vunomam", "Nazushagsaldôbar", "Sergeb", "Zafaldastot", "Vudnis", "Dostust", "Shotom", "Mugshith", "Angzak", "Oltud", "Deleratîs", "Nökornomal") //one of these is literally BLOOD POOL CREATE.iirc its Nazushagsaldôbar.
+
+/obj/item/smithing/hammerhead
+	name = "smithed hammer head"
+	finalitem = /obj/item/melee/smith/hammer
+	icon_state = "hammer"
+
+/obj/item/smithing/hammerhead/startfinish()
+	var/obj/item/melee/smith/hammer/finalforreal = new /obj/item/melee/smith/hammer(src)
+	finalforreal.force = ((finalforreal.force+quality/2)) * blunt_mult
+	finalforreal.qualitymod = quality/4
+	finalitem = finalforreal
+	..()
+
+
+
+/obj/item/smithing/scytheblade
+	name = "smithed scythe head"
+	finalitem = /obj/item/scythe/smithed
+	icon_state = "scythe"
+
+/obj/item/smithing/scytheblade/startfinish()
+	finalitem = new /obj/item/scythe/smithed(src)
+	finalitem.force = (finalitem.force+quality) * sharp_mult
+	..()
+
+/obj/item/smithing/shovelhead
+	name = "smithed shovel head"
+	finalitem = /obj/item/shovel/smithed
+	icon_state = "shovel"
+
+/obj/item/smithing/shovelhead/startfinish()
+	finalitem = new /obj/item/shovel/smithed(src)
+	finalitem.force = ((finalitem.force+quality)/2) * sharp_mult
+	if(quality > 0)
+		finalitem.toolspeed = max(0.05,(1-(quality/10)))
+	else
+		finalitem.toolspeed *= max(1, (quality * -1))
+	..()
+
+/obj/item/smithing/cogheadclubhead
+	name = "smithed coghead club head"
+	finalitem = /obj/item/melee/smith/cogheadclub
+	icon_state = "coghead"
+
+/obj/item/smithing/cogheadclubhead/startfinish()
+	finalitem = new /obj/item/melee/smith/cogheadclub(src)
+	finalitem.force = (finalitem.force+quality) * blunt_mult
+	..()
+
+/obj/item/smithing/javelinhead
+	name = "smithed javelin head"
+	finalitem = /obj/item/melee/smith/twohand/javelin
+	icon_state = "javelin"
+
+/obj/item/smithing/javelinhead/startfinish()
+	var/obj/item/melee/smith/twohand/javelin/finalforreal = new /obj/item/melee/smith/twohand/javelin(src)
+	finalforreal.force = (finalforreal.force+quality) * sharp_mult
+	finalforreal.wield_force = finalforreal.force*finalforreal.wielded_mult
+	finalforreal.AddComponent(/datum/component/two_handed, force_unwielded=finalforreal.force, force_wielded=finalforreal.wield_force, icon_wielded="[icon_state]")
+	finalforreal.throwforce = finalforreal.force*2
+	finalitem = finalforreal
+	..()
+
+/obj/item/smithing/pikehead
+	name = "smithed pike head"
+	finalitem = /obj/item/melee/smith/twohand/pike
+	icon_state = "pike"
+
+/obj/item/smithing/pikehead/startfinish()
```

Comment:
put spaces between the operators in anything else here, cannot be asssed to manually point out all thirty instances